Khan/perseus
# [Storybook] Update styles to be more accurate to prod

## Summary:
Perseus Storybook in its current state does not accurately
reflect how these component look in prod.

A couple things are added here to fix this:
1. Stories are wrapped with the `framework-perseus` class.
2. Webapp styles are imported
  - `box-sizing: border-box` in particular affects all the
    spacing (margins and padding)

Issue: none

## Test plan:
- Go to http://localhost:6006/?path=/docs/perseus-components-number-input--docs
- Confirm that the inputs have rounded bordersa, and the
  "mini", "small", and "normal" stories show different sizes.
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--interactive-graph-ray
- Scroll to the start coordinates panels
- Use the dev tools to confirm that the width is 308px
  (previously 336px)

## Screenshots:

| Before | After |
| --- | --- |
| <img width="299" alt="Screenshot 2024-12-03 at 6 03 03 PM" src="https://github.com/user-attachments/assets/ebe941b1-142a-4760-ad8e-397eabdb85a3">| <img width="286" alt="Screenshot 2024-12-03 at 6 02 37 PM" src="https://github.com/user-attachments/assets/382467fb-0bb3-4c59-b5b5-9f8a4698727f"> | 
| <img width="387" alt="Screenshot 2024-12-03 at 6 02 56 PM" src="https://github.com/user-attachments/assets/a3465089-548f-49af-842c-c2d87770c240"> | <img width="363" alt="Screenshot 2024-12-03 at 6 02 47 PM" src="https://github.com/user-attachments/assets/7ecba6c5-5468-400b-8150-4a733b870e0c"> |
